packages/dds/map/package.json

```diff
@@ -96,6 +96,43 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
```

Comment:
Can you check why these tests are broken? I don't see breaking changes in 1.2.0: https://github.com/microsoft/FluidFramework/pull/11025/files#diff-ca95ce2077e008bf603122e069ee1da158095cf74681aa5cac14657d83a97376